Core/BeamParams.h

```diff
@@ -25,33 +25,18 @@ class BeamParams
 {
 public:
   BeamParams();
-  void SetEnergy(G4double x)
-  {
-    energy = x * keV;
-  };
+  void SetEnergy(G4double x) {energy = x * keV;};
   void SetVtxPosition(G4double x, G4double y, G4double z);
   void SetPolarization(G4double x, G4double y, G4double z);
   void SetMomentum(G4double x, G4double y, G4double z);
   void SetVtxPosition(G4ThreeVector v);
   void SetPolarization(G4ThreeVector v);
   void SetMomentum(G4ThreeVector v);
 
-  G4ThreeVector GetVtx()
-  {
-    return vtxPosition;
-  };
-  G4double GetEne()
-  {
-    return energy;
-  };
-  G4ThreeVector GetMomentum()
-  {
-    return momentum;
-  };
-  G4ThreeVector GetPolarization()
-  {
-    return polarization;
-  };
+  G4ThreeVector GetVtx() {return vtxPosition;};
```

Comment:
All those functions can be const
e.g.` G4ThreeVector GetVtx() const {return vtxPosition;}`

See:
https://www.geeksforgeeks.org/const-member-functions-c/

Reply:
I think you also don't need the second ; after declaration

Reply:
This change would be good to make with the commit that will make one style of 
formatting. Before the relase of the new version. I will write it somewhere.

Reply:
*I meant comment about second ; after declaration